src/main/java/frc/robot/subsystems/RobotPoses.java

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.epilogue.Logged;
+import edu.wpi.first.math.geometry.Pose3d;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.math.geometry.Rotation3d;
+import edu.wpi.first.math.geometry.Transform3d;
+import edu.wpi.first.math.geometry.Translation3d;
+import edu.wpi.first.units.Units;
+import frc.robot.Robot;
+
+@Logged
+
+public class RobotPoses extends SubsystemBase {
+
+  Pose3d elevatorStageOne = Pose3d.kZero;
+  Pose3d elevatorCarriage = Pose3d.kZero;
+  Pose3d algaeIntake = Pose3d.kZero;
+  private Elevator subElevator;
+  private AlgaeIntake subAlgaeIntake;
+
```

Comment:
We should add @NotLogged (and import it) so that we are not double logging the subsystem
```suggestion
  @NotLogged
  private Elevator subElevator;
  @NotLogged
  private AlgaeIntake subAlgaeIntake;

```